pkg/utils/helper.go

```diff
@@ -0,0 +1,74 @@
+package utils
+
+import (
+	"encoding/json"
+	"fmt"
+	"gopkg.in/yaml.v2"
+)
+
+// function to get the relevant information out of k8s object that is needed to compute hash
+func GetHashObjectOfKubernetes(object interface{}) (map[string]interface{}, error) {
```

Comment:
This name tripped me up, I read it as `GetHashOfKubernetesObject` at first, would something like `GetKubernetesObjectForHashing` work as a name? I think it just seems non-obvious what a `HashObject` is  exactly: the name sounds like it's an object that represents a hash.

Reply:
Yeah you are right. Bad naming skills :(